src/vos/vos_pool.c

```diff
@@ -598,8 +598,8 @@ vos_pmemobj_create(const char *path, uuid_t pool_id, const char *layout,
 	} else if ((scm_sz) && (meta_sz != scm_sz)) {
 		// See below comment on DAV allocator support and remove check when completed.
 		D_ERROR("scm_size != meta_size pool create case not supported, scm_sz: " DF_U64
-			" meta_sz: " DF_U64, scm_sz, meta_sz);
-		return -DER_INVAL;
+			" meta_sz: " DF_U64,
+			scm_sz, meta_sz);
```

Comment:
An existing unit test passes 2 for 'meta_sz'? Isn't the 'meta_sz' parameter added by this PR?

Reply:
Please return error for this unsupported case.